logger.go

```diff
@@ -281,6 +292,43 @@ func (l *Logger) Fatalf(format string, v ...interface{}) {
 	os.Exit(1)
 }
 
+// Verbose is a boolean type that implements Infof, etc.
+type Verbose bool
+
+// V generates a log record depends on the setting of the Level; or none default.
+func V(lvl Level) Verbose {
```

Comment:
You also need to create these for Logger, you just created them for the default logger

Reply:
Done.